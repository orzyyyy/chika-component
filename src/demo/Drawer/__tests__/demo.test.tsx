import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import DrawerDemo from '..';

describe('DrawerDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<DrawerDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
