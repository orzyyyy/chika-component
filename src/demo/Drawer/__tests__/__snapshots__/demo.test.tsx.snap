// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerDemo render correctly 1`] = `
<div
  className="Drawer_demo"
>
  <Drawer
    direction="left"
    onChange={[Function]}
    visible={false}
  >
    test
  </Drawer>
</div>
`;
